### PR TITLE
Make jenkins e2e runs reuse one SSH key.

### PR DESCRIPTION
**Pull request by mbforbes**
Helps alleviate #7714; other part of fix coming soon.

This will stop us from filling up the project with keys and cycling them.

+cc @ixdy @quinton-hoole

### CODE REVIEW

File: hack/jenkins/e2e.sh [hunk 1]
```diff
@@ -34,6 +34,11 @@ if [[ "${CIRCLECI:-}" == "true" ]]; then
     WORKSPACE=`pwd`
 else
     # Jenkins?
+    # Copy GCE keys so we don't keep cycling them.
+    mkdir -p ${WORKSPACE}/.ssh/
+    cp /var/lib/jenkins/gce_keys/google_compute_engine ${WORKSPACE}/.ssh/
```
Comment (ixdy) [inline hack/jenkins/e2e.sh]: Where did these come from? Might be worth documenting the setup somewhere.

Comment (mbforbes) [inline hack/jenkins/e2e.sh]: I added a string of comments above it about the setup. Does this look all right?

**Comment by mbforbes** [comment]
ping @zmerlynn

**Comment by zmerlynn** [comment]
Oh, what the hell. Friday afternoon is a great time to kill Jenkins. LGTM.

**Comment by saad-ali** [comment]
This is breaking Jenkins.

It sets up the .ssh folder in the workspace as the first step.
`hack/jenkins/e2e.sh` checks to make sure that the workspace is empty and fails if it is not:
```
+ echo /jenkins-master-data/jobs/kubernetes-e2e-gce/workspace not empty, 'bailing!'
/jenkins-master-data/jobs/kubernetes-e2e-gce/workspace not empty, bailing!
+ exit 1
Build step 'Execute shell' marked build as failure
Recording test results
ERROR: Publisher hudson.tasks.junit.JUnitResultArchiver aborted due to exception
hudson.AbortException: No test report files were found. Configuration error?

```

**Comment by mbforbes** [comment]
Yep, we're on it.

**Comment by saad-ali** [comment]
Great, thanks.

**Pull request merged**